iceoryx2-bb/lock-free/tests/spmc_unrestricted_atomic_tests.rs

```diff
@@ -164,3 +169,104 @@ fn spmc_unrestricted_atomic_get_ptr_write_and_update_works_concurrently() {
         pred = v;
     }
 }
+
+#[test]
+fn spmc_unrestricted_atomic_mgmt_release_producer_allows_new_acquire() {
+    let _test_lock = TEST_LOCK.lock().unwrap();
+    let sut = UnrestrictedAtomicMgmt::new();
+
+    let p1 = unsafe { sut.__internal_acquire_producer() };
+    assert_that!(p1, is_ok);
+    assert_that!(unsafe { sut.__internal_acquire_producer() }, is_err);
+
+    unsafe {
+        sut.__internal_release_producer();
+    }
+
+    let p2 = unsafe { sut.__internal_acquire_producer() };
+    assert_that!(p2, is_ok);
+}
+
+#[test]
+fn spmc_unrestricted_atomic_mgmt_get_ptr_write_and_update_works() {
+    let _test_lock = TEST_LOCK.lock().unwrap();
+
+    let value: u64 = 3;
+    let value_ptr: *const u64 = &value;
+    let value_size = core::mem::size_of::<u64>();
+    let value_alignment = core::mem::align_of::<u64>();
+
+    let mut read_value: u64 = 0;
+    let read_value_ptr: *mut u64 = &mut read_value;
+
+    let atomic = UnrestrictedAtomic::<u64>::new(0);
```

Comment:
Could you please put the `0` into a variable called `const INITIAL_VALUE: u64 = 0;` and use it in line 219. This makes the unit test more understandable.